src/components/MultiHopTrade/components/TradeInput/components/ManualAddressEntry.tsx

```diff
@@ -57,21 +62,30 @@ export const ManualAddressEntry: FC<ManualAddressEntryProps> = memo(
     )
     const { manualReceiveAddress } = useReceiveAddress(useReceiveAddressArgs)
 
+    const isAccountMetadataLoading = useAppSelector(selectIsAccountMetadataLoading)
+
     const shouldShowManualReceiveAddressInput = useMemo(() => {
-      if (isAccountMetadataLoading) return false
+      // Some AccountIds are loading for that chain - don't show the manual address input since these will eventually be populated
+      if (
+        Object.entries(isAccountMetadataLoading).some(
+          ([accountId, isLoading]) => fromAccountId(accountId).chainId === chainId && isLoading,
+        )
+      )
+        return false
```

Comment:
suggestion: we could verify that sellAsset chainId is actually loading, it would be more granular, but it might create some side effects, if it's simpler I'm happy to keep it as it is